packages/create-pwa/lib/index.js

```diff
@@ -147,40 +153,44 @@ module.exports = async () => {
     let answers;
     try {
         answers = await inquirer.prompt(questions);
-    } catch (e) {
-        console.error('App creation cancelled.');
-    }
-    answers.backendUrl = answers.backendUrl || answers.customBackendUrl;
-    const args = questions.reduce(
-        (args, q) => {
-            if (q.name === 'customBackendUrl' || q.name === 'directory') {
-                return args;
-            }
-            const answer = answers[q.name];
-            const option = changeCase.paramCase(q.name);
-            if (q.type === 'confirm') {
-                if (answer !== q.default) {
-                    return [...args, answer ? `--${option}` : `--no-${option}`];
+
+        answers.backendUrl = answers.backendUrl || answers.customBackendUrl;
+        const args = questions.reduce(
+            (args, q) => {
+                if (q.name === 'customBackendUrl' || q.name === 'directory') {
+                    return args;
                 }
-                return args;
-            }
-            return [...args, `--${option}`, `"${answer}"`];
-        },
-        ['create-project', answers.directory, '--template', '"venia-concept"']
-    );
+                const answer = answers[q.name];
+                const option = changeCase.paramCase(q.name);
+                if (q.type === 'confirm') {
+                    if (answer !== q.default) {
+                        return [
+                            ...args,
+                            answer ? `--${option}` : `--no-${option}`
+                        ];
+                    }
+                    return args;
+                }
+                return [...args, `--${option}`, `"${answer}"`];
+            },
+            ['create-project', answers.directory]
+        );
 
-    const argsString = args.join(' ');
+        const argsString = args.join(' ');
 
-    console.log(
-        '\nRunning command: \n\n' +
-            chalk.whiteBright(`buildpack ${argsString}\n\n`)
-    );
+        console.log(
+            '\nRunning command: \n\n' +
+                chalk.whiteBright(`buildpack ${argsString}\n\n`)
+        );
 
-    const buildpackBinLoc = resolve(
-        require.resolve('@magento/pwa-buildpack'),
-        '../../bin/buildpack'
-    );
-    await execa.shell(`${buildpackBinLoc} ${argsString}`, {
-        stdio: 'inherit'
-    });
+        const buildpackBinLoc = resolve(
+            require.resolve('@magento/pwa-buildpack'),
+            '../../bin/buildpack'
+        );
+        await execa.shell(`${buildpackBinLoc} ${argsString}`, {
+            stdio: 'inherit'
+        });
+    } catch (e) {
```

Comment:
This is just a small change. Moved catch statement to the end of the function.

Reply:
Thanks for the call out! You can see this by appending `?w=1` :) https://github.com/magento/pwa-studio/pull/3025/files?w=1

Reply:
Good to know 👍